lib/util/http/queue.ts

```diff
@@ -15,7 +18,10 @@ export function getQueue(url: string): PQueue | null {
     queue = null; // null represents "no queue", as opposed to undefined
     const concurrency = getRequestLimit(url);
     if (concurrency) {
+      logger.debug({ concurrency, host }, 'Using queue');
       queue = new PQueue({ concurrency });
+    } else {
+      logger.debug({ host }, 'No concurency limits');
```

Comment:
Are these going to log a debug message for every single http request?

Reply:
nope, once per host